back/app/controllers/web_api/v1/cosponsors_initiatives_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class WebApi::V1::CosponsorsInitiativesController < ApplicationController
+  def accept_invite
+    @cosponsors_initiative = CosponsorsInitiative.find(params[:id])
+    authorize @cosponsors_initiative, :accept_invite?
+
+    if @cosponsors_initiative.update(status: 'accepted')
+      render json: WebApi::V1::CosponsorsInitiativeSerializer.new(
+        @cosponsors_initiative,
+        params: jsonapi_serializer_params
+      ).serializable_hash
+    end
```

Comment:
```suggestion
    @cosponsors_initiative.update!(status: 'accepted')
    head :ok
```
What is returned if update fails? To avoid this ambiguity, we could use `!` (though it should never raise an exception atm).

If we don't use this serializer in other places, we don't need to introduce it.

Reply:
We don't use the serializer anywhere else atm.